app/goal_routes.py

```diff
@@ -0,0 +1,101 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.task import Task
+from app.models.goal import Goal
+from app.task_routes import validate_task
+from app import db
+
+goal_bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+def validate_goal(goal_id):
+    try:
+        goal_id = int(goal_id)
+    except:
+        abort(make_response({"message":f"Goal {goal_id} invalid"}, 400))
+    
+    goal = Goal.query.get(goal_id)
+    
+    if not goal:
+        abort(make_response({"message":f"Goal {goal_id} not found"}, 404))
+    return goal if goal else abort(make_response({"message":f"Goal {goal_id} not found"}, 404))
+
+#CREATE GOALS
+
+@goal_bp.route("/<goal_id>/tasks", methods=["POST"])
+def create_goal(goal_id):
+    request_body = request.get_json()
+    try:
+        new_goal = Goal.from_dict(request_body)
+        new_goal.description == "Test Description"
+        new_goal.completed_at == None
```

Comment:
Hm, what are these lines doing? To me, it reads that you're checking if `new_goal.description` is equal to `"Test Description"`, and checking if `new_goal.completed_at` is equal to `None`. Is that what you were intending?